Library/Formula/h264enc.rb

```diff
@@ -0,0 +1,76 @@
+class H264enc < Formula
+  homepage "http://h264enc.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/h264enc/h264enc/h264enc-9.9.0.tar.gz"
+  sha1 "8bfa7017386a4b7612acd21e48dedf940652582d"
+
+  # depends_on "cmake" => :build
+  # depends_on :x11 # if your formula requires any X11/XQuartz components
+  # [encode, x264], mplayer needs x264 installed first 
```

Comment:
mplayer will actually be installed as a binary package for most users, so it won't opportunistically pick up undeclared libraries in most cases.

It'd probably be better to add libx264 as a dependency to mplayer, either by default or optionally, and let this formula depend on that. Ditto the other required dependencies.